pixi/src/ui/recommendations/RecommendationsView.js

```diff
@@ -39,8 +39,9 @@ export default class RecommendationsView {
       this.container.removeChild(items[i]);
     }
 
-    for (let i = 1; i < this.filter.children.length; i++) {
-      this.filter.removeChild(this.filterPills[i]);
+    const length = this.filter.children.length;
```

Comment:
Any chance filter would be undefined?

Reply:
Not unless it gets removed from the j2 fragment, but as it stands it's always defined on construction of this class.